simphony/testing/abc_check_engine.py

```diff
@@ -35,173 +39,186 @@ def create_dataset_items(self):
         """
 
     @abc.abstractmethod
-    def check_emtpy_dataset(self, ds):
-        """ Check if a dataset is empty
-        """
-
-    @abc.abstractmethod
     def check_instance_of_dataset(self, ds):
         """ Check if a dataset is instance of a class
         """
 
-    @abc.abstractmethod
-    def check_is_usable_dataset(self, ds, items):
-        """ Check if a dataset is usable
-        """
+    operation_mapping = {
+        'compare datasets': 'none',
+        'add item': 'none'}
+
+    def compare_operation(self, *args, **kwrds):
+        method = getattr(
+            sys.modules[__name__],
+            self.operation_mapping['compare datasets'])
+        return method(*args, **kwrds)
+
+    def add_operation(self, container, *args, **kwrds):
```

Comment:
Correct me if i am wrong, but I think this is unused and this method as well as the "add_items" entry in `operation_mapping` can be removed.